docs/bad_dump.md

```diff
@@ -0,0 +1,12 @@
+1. `ssh deploy@bibdata-alma-worker1`
+1. `cd /opt/marc_liberation/current`
+1. `RAILS_ENV=production bundle exec rails c`
+1. Find the IndexManager instance that uses the solr production collection. Currenlty is:`catalog-alma-production`
+`index_manager_current = IndexManager.all.first`
```

Comment:
We should update this. We need to find the current index manager which is in production. It might not be the first.